operations/github/ent-mfa/implementation_plan.md

```diff
@@ -0,0 +1,58 @@
+## Context
+OpenSSF [security baseline](https://github.com/ossf/tac/blob/main/process/security_baseline.md#security-baseline---once-sandbox) requires all technical initiative adopt 2FA in GitHub as part of the [TAC project lifecycle governance](https://github.com/ossf/tac/blob/main/process/project-lifecycle.md#sandbox). 
+This requires GitHub two-factor authentication(2FA) to be enabled in the enterprise account to enable 2FA by default for all the organizations and repositories. 
+
+Currently, 2FA is not enabled by default for a few organizations, event though maintainers have enabled 2FA in their accounts. 
+
+## Summary of Changes
+The plan is to: 
+* Enable authentication security setting: "Require two-factor authentication for all organizations in the Open Source Security Foundation enterprise" 
+
+As of now, GitHub does not provide an API to enable 2FA. This feature needs to be enabled manually through the GitHub user interface.
+
+## Prerequisites
+* The implementer needs to have GitHub Enterprise Owner role
+
+## Implementer
+Dana Wang
+
+## Implementation Time
+August 6, 2024 5PM EST
+
+## Impacts 
+
+The change will have ni impact to 224 of thw 225 members in various OpenSSF GitHub organization. 
+![no impact orgs](https://github.com/ossf/staff/blob/2543a62481f1b10b19a8c89f123847ef19723cb6/operations/github/ent-mfa/images/organizations_not_impacted.png)
```

Comment:
These image references seem to point to instances in a specific branch, rather than a relative link within the current branch.  Therefore, these may be deleted when this branch gets merged.  You likely want to add relative links.
e.g. `![no impact orgs](operations/github/ent-mfa/images/organizations_not_impacted.png)` or `![no impact orgs](images/organizations_not_impacted.png)`